win32ss/user/ntuser/timer.c

```diff
@@ -219,8 +221,12 @@ IntSetTimer( PWND Window,
   {
       IntLockWindowlessTimerBitmap();
 
-      IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex);
-
+      IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex++);
+      if (IDEvent == (UINT_PTR) -1)
```

Comment:
```suggestion
      if (IDEvent == (UINT_PTR)-1)
```
nitpick suggestion. Vast majority of our code tries to avoid a space before the subject of a cast. Rest of the PR LGTM.